### PR TITLE
Buttons

### LINKED ISSUE #235: Improve Course Info Buttons on SectionTable
Create a button component that we can use for the course section table buttons. Right now, they are very ugly links.
image

The button component should take in props for an icon, the text, and the redirect link.
For styling, I'm thinking we do a slightly rounded button, with the icon on the left and text on the right. 
For _reference_, here are mockups from Design at UCI. Idk if dark blue is the move though - feel free modify the design!
image

Create the following buttons
- `text`: "Prerequisites", `icon`:  maybe a warning icon ? (idk), `redirect link`: websoc (see existing link)
- `text` "Past Enrollment", `icon`: chart going up icon, `redirect link`: Zot Tracker (?)
- `text` "Previous Grades", `icon`: bar graph chart icon, `redirect link`: Zotistics (see existing link)

### PR DESCRIPTION
**Pull request by falsidge**
## Summary
Changed the links to material design buttons.
## Test Plan
Tested clicking buttons and they open correctly.
## Issues
#235

### CODE REVIEW

File: client/src/components/SectionTable/SectionTable.js [hunk 1]
```diff
@@ -82,42 +83,33 @@ class SectionTable extends PureComponent {
...
+                        size="small"
+                        onClick={(event) => {
+                            window.open(
+                                `https://zotistics.com/?&selectQuarter=&selectYear=&selectDep=${urlEncode}&classNum=${courseDetails.courseNumber}&code=&submit=Submit`
+                            );
+                        }}
+                        style={{ marginRight: '4px', backgroundColor: '#385EB1', color: '#fff' }}
+                    >
+                        {`Zotistics`}
+                    </Button>
```
Comment (ChaseC99) [inline client/src/components/SectionTable/SectionTable.js]: Let's abstract this to its own component called `CourseInfoButton`.

Please refer to the issue for what the 3 props are.

**Review by ChaseC99 — CHANGES_REQUESTED** [review]

File: client/src/components/SectionTable/SectionTable.js (hunk 1 above)
Comment (falsidge) [reply client/src/components/SectionTable/SectionTable.js]: Sorry had to force push rebasing branch. I made the buttons into components in the latest commit.

File: client/src/components/SectionTable/CourseInfoButton.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@material-ui/core';
+
+class CourseInfoButton extends PureComponent {
+    render() {
+        const { text, icon, redirectLink } = this.props;
+        return (
+            <Button
+                endIcon={icon}
```
Comment (ChaseC99) [inline client/src/components/SectionTable/CourseInfoButton.js]: Let's move the icon to the left side
```suggestion
                startIcon={icon}
```

File: client/src/components/SectionTable/SectionTable.js [hunk 3]
```diff
@@ -82,42 +84,19 @@ class SectionTable extends PureComponent {
...
-                                }}
-                                href={courseDetails.prerequisiteLink}
-                                rel="noopener noreferrer"
-                            >
-                                Prerequisites
-                            </a>
-                        </Typography>
+                        <CourseInfoButton
+                            text="Prerequisites"
+                            icon={<PriorityHigh />}
```
Comment (ChaseC99) [inline client/src/components/SectionTable/SectionTable.js]: Hmm actually I'm not a huge fan of the `!` 

Maybe we should use this instead? https://mui.com/components/material-icons/?selected=TaskAlt
image


Or here's a few other suggestions:
- https://mui.com/components/material-icons/?selected=GppGood
- https://mui.com/components/material-icons/?selected=FileDownloadDone
- https://mui.com/components/material-icons/?selected=Merge

**Review by ChaseC99 — CHANGES_REQUESTED** [review]
Looking awesome! Just a few nits and it should be good to go

File: client/src/components/SectionTable/SectionTable.js (hunk 3 above)
Comment (falsidge) [reply client/src/components/SectionTable/SectionTable.js]: Those icons aren't in the @material-ui/icons library. Assignment might look good too.

https://mui.com/components/material-icons/?selected=Assignment

image

Comment (ChaseC99) [reply client/src/components/SectionTable/SectionTable.js]: Ah bummer but yeah I think Assignment does look good! Let's go with that.

**Review by ChaseC99 — APPROVED** [review]
Those are so good looking buttons 🔥 
Great work!

**Pull request merged**